src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -88,12 +88,9 @@ def _print_examples(help_file):
             print('')
 
     @staticmethod
-    def _print_az_find_message(command, enable_color):
-        from colorama import Style
+    def _print_az_find_message(command):
         indent = 0
         message = 'To search AI knowledge base for examples, use: az find "az {}"'.format(command)
-        if enable_color:
-            message = Style.BRIGHT + message + Style.RESET_ALL
```

Comment:
We remove the color here as it outshines whole original help messages and examples.